### PR TITLE
External announcer

### PR DESCRIPTION
**Pull request by ghost**
The external announcer can be tested as per: https://github.com/TwitchPlaysPokemon/tpp/pull/485

### CODE REVIEW

File: pbrEngine/engine.py [hunk 1]
```diff
@@ -2012,6 +2057,7 @@ def _distinguishAttack(self, data):
                            slot=slot,
                            moveindex=0,  # FIXME or remove me
                            movename=moveName,
+                           success=self._nextMoveSuccess,
```
Comment (Felk) [inline pbrEngine/engine.py:2058]: what is the expected use for this?

File: pbrEngine/engine.py [hunk 2]
```diff
@@ -157,11 +155,26 @@ def _crash(reason=None):
               CAUTION: broken- needs fixing or removal
         arg3: <movename> name of the move used.
               CAUTION: The attacking pkmn might not have this attack (e.g. Metronome)
+        arg4: <success> whether the move is successful.
+              I.e., it doesn't miss or fail.  For a move with multiple targets, this is 
+              true if the move succeeds against at least one target.
         arg4: <obj> object originally returned by the action-callback that led
```
Comment (Felk) [inline pbrEngine/engine.py]: arg5, and this documentation was missing "teams" before too.

**Review by Felk — COMMENTED** [review]
did you intend to commit those two binary files regarding type effectivenesses?

File: pbrEngine/engine.py (hunk 2 above)
Comment (ghost) [reply pbrEngine/engine.py]: I need to update documentation on a few of the callbacks, do you want those changes in this PR or a new one?

File: pbrEngine/engine.py (hunk 1 above)
Comment (ghost) [reply pbrEngine/engine.py:2058]: An external announcer watches the attack callback and plays one line calling out the move if it succeeded, or a different line calling out a miss if it failed.

**Comment by ghost** [comment]
> 
> 
> did you intend to commit those two binary files regarding type effectivenesses?

Yes, they're just part of the April Fools changes.  I could put them in another PR, or simply call this PR "April Fool's changes" or something- unless you think alternate effectiveness type chart binaries belong in the tpp repository rather than pbrengine?

File: pbrEngine/engine.py (hunk 2 above)
Comment (Felk) [reply pbrEngine/engine.py]: however you prefer it, I don't mind either

**Review by Felk — APPROVED** [review]

File: pbrEngine/engine.py (hunk 2 above)
Comment (ghost) [reply pbrEngine/engine.py]: I will make them in a separate PR this weekend

**Pull request merged**